ansible/uod-docker.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Playbook for managing a docker instance in the
+# openstack cloud.
+
+- hosts: docker-hosts
+  vars:
+    ansible_user: centos
+  roles:
+  - role: storage-volume-initialise
+    storage_volume_initialise_device: /dev/vdb
+    storage_volume_initialise_mount: /var/lib/docker
+  - role: basedeps
+  - role: system-monitor-agent
+  - role: versioncontrol-utils
+  - role: docker
+    docker_groupmembers: [centos]
+  - role: sudoers
+  - role: local-accounts
```

Comment:
@joshmoore - giving the vagrant VM more ram is enough to let the playbook get past the systemd startup stage - this failed for me the first time I tried vagrant provisioning, too.

```
LS27172:ansible kenny$ git diff -p
diff --git a/ansible/Vagrantfile b/ansible/Vagrantfile
index 73e6cf2..1b4a5db 100644
--- a/ansible/Vagrantfile
+++ b/ansible/Vagrantfile
@@ -5,6 +5,12 @@
 # See: http://docs.ansible.com/ansible/guide_vagrant.html
 Vagrant.require_version ">= 1.7.0"

+Vagrant.configure("2") do |config|
+  config.vm.provider "virtualbox" do |vb|
+    vb.customize ["modifyvm", :id, "--memory", "1024"]
+  end
+end
+
 Vagrant.configure(2) do |config|

   config.vm.define "docker" do |docker|
```